app/controllers/provider_foreman_controller.rb

```diff
@@ -982,6 +1028,25 @@ def add_unassigned_configuration_profile_record_to_view(unassigned_profile_row,
     @grid_hash = view_to_hash(@view)
   end
 
+  def empty_inventory_group_record?(inventory_group_record)
+    inventory_group_record.try(:id).nil?
+  end
+
+  def valid_inventory_group_record?(inventory_group_record)
+    inventory_group_record.try(:id)
+  end
+
+  def inventory_group_right_cell_text(model)
+    record_model = ui_lookup(:model => model || TreeBuilder.get_model_for_prefix(@nodetype))
```

Comment:
Should this line be inside the `if` a couple lines below?  Then we would only do the lookup if we plan to use it.